packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -2825,3 +2837,223 @@ test('Bug 7784 reconstitute both', async t => {
     },
   });
 });
+
+// This test fails if `auctionDriver.capturePrices()` is moved from line 253 in
+// https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/src/auction/scheduler.js
```

Comment:
thanks for this comment. the line numbers will be stale wrt master. consider using the permalink, https://github.com/Agoric/agoric-sdk/blob/bf4b59f2128aca73d786429c6ef4a7c9cdfb6700/packages/inter-protocol/src/auction/scheduler.js#L253

Reply:
Oh! thanks. I thought I was copying a permalink.